tests/unit/test_steve.py

```diff
@@ -108,3 +108,20 @@ def test_process_message(event, private, enabled_private_namespaces, success):
     )
     assert "propose_downstream" in next(iter(results["job"]))
     assert first_dict_value(results["job"])["success"]
+
+
+@pytest.fixture()
+def github_push():
+    with open(DATA_DIR / "webhooks" / "github" / "push.json") as outfile:
```

Comment:
We already have an example. Nice!

Reply:
that's also a problem :D it's copy-paste, would be nice to do a PR and reorganize all those fixtures

Reply:
Would be really nice!